flaky test TestInterruptedDuringSpilling

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)

```
 === RUN   TestInterruptedDuringSpilling
[2024/01/30 01:31:42.297 +00:00] [INFO] [sort_partition.go:255] ["memory exceeds quota, spill to disk now."] [consumed=935444480] [quota=-1]
    sortexec_pkg_test.go:129: 
        	Error Trace:	pkg/executor/sortexec/sortexec_pkg_test.go:129
        	Error:      	"1.02931491s" is not less than "1s"
        	Test:       	TestInterruptedDuringSpilling
--- FAIL: TestInterruptedDuringSpilling (43.16s) 
```
### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->
Automated flaky test report update.

- Repo: pingcap/tidb
- Branch: master
- Package: pkg/executor/sortexec
- Case: TestInterruptedDuringSpilling
- Flaky Count: 3
- Time Thresholded Count: 3
- Latest Build: https://prow.tidb.net/jenkins/job/pingcap/job/tidb/job/pull_unit_test_next_gen/551/

Window: 2026-04-12T00:00:00.000Z → 2026-04-13T00:00:00.000Z
Threshold: 100000 ms
Reopen information⬆️

--- a/pkg/executor/sortexec/BUILD.bazel
+++ b/pkg/executor/sortexec/BUILD.bazel
@@ -31,23 +31,8 @@ go_library(
 )
 
 go_test(
-    name = "sort_exec_test",
-    timeout = "short",
-    srcs = ["sort_test.go"],
-    flaky = True,
-    shard_count = 6,
-    deps = [
-        "//pkg/config",
-        "//pkg/sessionctx/variable",
-        "//pkg/testkit",
-        "//pkg/util",
-        "@com_github_pingcap_failpoint//:failpoint",
-        "@com_github_stretchr_testify//require",
-    ],
-)
-
-go_test(
     name = "sortexec_test",
+    timeout = "short",
     srcs = [
         "benchmark_test.go",
         "sort_spill_test.go",
@@ -55,6 +40,8 @@ go_test(
         "sortexec_pkg_test.go",
     ],
     embed = [":sortexec"],
+    flaky = True,
+    shard_count = 6,
     deps = [
         "//pkg/config",
         "//pkg/executor/internal/exec",

